packages/generator/src/generator-utils.ts

```diff
@@ -185,23 +189,37 @@ export function getGenericParameters(
   entityClassName: string,
   prop: VdmProperty
 ): string {
-  const param = prop.isCollection
-    ? prop.isComplex
-      ? [`${prop.jsType}`]
-      : [`'${prop.edmType}'`]
-    : [];
+  let param;
```

Comment:
Minor: You can initialize with [] here and get rid of the `else` black later on.